### PR TITLE
Update windows.md

### PR DESCRIPTION
**Pull request by embray**
With apologies (it wasn't clear from the contributing guidelines how this project feels about PRs for one or two word doc fixes).

### CODE REVIEW

**Comment by jhowardmsft** [comment]
LGTM. Just needs the right signing on the PR.

**Comment by embray** [comment]
Signed. (I was just in the process, but wasn't sure at first if that was needed for trivial doc fixes)

File: docs/installation/windows.md [hunk 1]
```diff
@@ -87,12 +87,12 @@ to your console window and you are ready to run docker commands such as
...
 
-Launch a PowerShell window, then you need to add `ssh.exe` to your PATH:
+Launch a PowerShell window, then add `ssh.exe` to your PATH:
 
     $Env:Path = "${Env:Path};c:\Program Files (x86)\Git\bin"
 
-and after running `boot2docker start` command it will print PowerShell commands
-to set the environment variables to connect Docker running inside VM. Run these
+and after running the `boot2docker start` command it will print PowerShell commands
+to set the environment variables to connect to Docker running inside VM. Run these
```
Comment (thaJeztah) [inline docs/installation/windows.md]: while we're here; can you change "inside VM" to "inside the VM"?

**Comment by thaJeztah** [comment]
thanks so much @embray. Yes we switched to requiring sign-offs for all contributions, also trivial ones a while ago (sorry for the hassle)

I added one note, but LGTM even without that. Let us know if you feel like addressing that one too :smile:

File: docs/installation/windows.md (hunk 1 above)
Comment (embray) [inline docs/installation/windows.md]: Sure, yeah, I thought that looked awkward.  I'll just add that and squash.

Comment (embray) [inline docs/installation/windows.md]: Done.

**Comment by thaJeztah** [comment]
LGTM! \o/

ping @fredlf @jamtur01 @moxiegirl @SvenDowideit

File: docs/installation/windows.md [hunk 2]
```diff
@@ -87,12 +87,12 @@ to your console window and you are ready to run docker commands such as
...
 
-Launch a PowerShell window, then you need to add `ssh.exe` to your PATH:
+Launch a PowerShell window, then add `ssh.exe` to your PATH:
 
     $Env:Path = "${Env:Path};c:\Program Files (x86)\Git\bin"
 
-and after running `boot2docker start` command it will print PowerShell commands
-to set the environment variables to connect Docker running inside VM. Run these
+and after running the `boot2docker start` command it will print PowerShell commands
+to set the environment variables to connect to Docker running inside the VM. Run these
```
Comment (jamtur01) [inline docs/installation/windows.md]: to the Docker daemon running inside ... 

Perhaps?

Comment (embray) [inline docs/installation/windows.md]: :+1:

**Comment by thaJeztah** [comment]
ah, even better.

@jamtur01 your comment was addressed, so all yours to merge if it lgty

**Comment by jamtur01** [comment]
LGTM

**Comment by moxiegirl** [comment]
cherry-picked

**Pull request merged**